jupyter-pytorch-full/tests/test_imports.py

```diff
@@ -13,14 +13,12 @@
 def main():
     """Test running container and imports."""
     rock = yaml.safe_load(Path("rockcraft.yaml").read_text())
-    name = rock["name"]
+    rock_image = rock["name"]
     rock_version = rock["version"]
-    arch = list(rock["platforms"].keys())[0]
-    rock_image = f"{name}_{rock_version}_{arch}"
     LOCAL_ROCK_IMAGE = f"{rock_image}:{rock_version}"
     
     print(f"Running command in {LOCAL_ROCK_IMAGE}")
-    script_command = ["bash", "-c", "export PYTHONPATH=$PYTHONPATH:/opt/conda/lib/python3.8/site-packages/keras/api/keras/wrappers/:/opt/conda/lib/python3.8/site-packages/ && python3 /home/jovyan/imports.py"]
+    script_command = ["bash", "-c", "export HOME=/home/jovyan && export PATH=/opt/conda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin && PYTHONPATH=/opt/conda/lib/python3.11/site-packages/  python3.11 /home/jovyan/imports.py"]
```

Comment:
I 'm wondering why we need to update this test here since the PR doesn't edit the rock's command? Do you know if this test worked before?

Reply:
This is needed as we need to setup the environment the same way as we do for command ... not sure if the previous thing was even working